.github/workflows/backend.yml

```diff
@@ -19,17 +19,38 @@ jobs:
   files-changed:
     name: Check which files changed
     runs-on: ubuntu-22.04
-    timeout-minutes: 3
+    timeout-minutes: 6
     outputs:
       backend_all: ${{ steps.changes.outputs.backend_all }}
+      static_viz: ${{ steps.static_viz.outputs.static_viz }}
     steps:
       - uses: actions/checkout@v3
+      - name: Prepare front-end environment
+        uses: ./.github/actions/prepare-frontend
+      - name: Prepare back-end environment
+        uses: ./.github/actions/prepare-backend
+        with:
+          m2-cache-key: "files-changed"
+      - run: yarn install --frozen-lockfile --prefer-offline
```

Comment:
Why do we need to install packages **again**?
This command is already executed as a part of `Prepare front-end environment` action.

https://github.com/metabase/metabase/blob/master/.github/actions/prepare-frontend/action.yml#L28

Reply:
Good catch! I'll fix that